src/tests/integration/configuration-example.json

```diff
@@ -1,22 +1,63 @@
 {
   "local_test": true,
   "api": {
-    "local_base_url": "https://localhost:8080/api/v1",
-    "base_url": "https://api.oregonstate.edu/v1"
+    "local_base_url": "https://localhost:8080/api/v2",
+    "base_url": "https://api.oregonstate.edu/v2"
   },
   "auth": {
     "basic_auth": {
       "username": "username",
       "password": "password"
-    },
-    "oauth2": {
-      "token_api_url": "https://api.oregonstate.edu/oauth2/token",
-      "client_id": "client_id",
-      "client_secret": "client_secret"
     }
   },
   "test_cases": {
-    "valid_pet_ids": ["1", "2", "3"],
-    "invalid_pet_ids": ["999", "-1", "invalid_id"]
+    "id": {
+      "valid": ["43868700139", "53868730879", "44979934575", "28313180078"],
+      "invalid": ["-2", "0", "noID"]
+    },
+    "firstName": {
+      "valid": ["Frederick", "Toby", "Greta"],
+      "invalid": ["fdsafdsa", "12357"]
+    },
+    "lastName": {
+      "valid": ["McDonald", "Liang", "Wilson"],
+      "invalid": ["fdsafdsa", "12357"]
+    },
+    "onid": {
+      "valid": ["kalcicd", "awaik", "kauffmza"],
+      "invalid": ["123", "asdf"]
+    },
+    "primaryAffiliation": {
+      "valid": ["Employee", "Student", "Other"],
+      "invalid": ["1234", "fdsa", "djklsa"]
+    },
+    "emailAddress": {
+      "valid": ["kalcicd@oregonstate.edu", "awaik@oregonstate.edu", "kauffmza@oregonstate.edu"],
+      "invalid": ["1234", "fdsa", "djklsa"]
+    },
+    "phoneNumber": {
+      "valid": ["15418670386", "0386", "404"],
+      "invalid": ["asdf", "noNumbersHere123"]
+    },
+    "alternatePhoneNumber": {
+      "valid": [],
+      "invalid": []
+    },
+    "officePhoneNumber": {
+      "valid": [],
+      "invalid": []
+    },
+    "faxNumber": {
+      "valid": [],
+      "invalid": []
+    },
+    "officeAddress": {
+      "valid": [],
+      "invalid": []
+    },
```

Comment:
There are no test cases for these because the query parameters are bugged. See [this](https://jira.sig.oregonstate.edu/browse/CO-1634) ticket. Will add test cases once the issues are resolved so the tests can be run without erroring.

Reply:
Should be a comment in the code instead of github

Reply:
added a comment above the test function